internal/util/cephcmds.go

```diff
@@ -32,6 +32,38 @@ import (
 // InvalidPoolID used to denote an invalid pool.
 const InvalidPoolID int64 = -1
 
+func ExecCommandWithSenter(ctx context.Context, netPath, program string, args ...string) (string, string, error) {
```

Comment:
name of the function should probably be `ExecuteCommandWithNSEnter` (missing `N`).